e2e/e2e-windows-scenario.sh

```diff
@@ -101,6 +101,7 @@ WINDOWS_PASSWORD=$({
     choose '0123456789'
     choose 'abcdefghijklmnopqrstuvwxyz'
     choose 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    choose '#*'
```

Comment:
this may still do not work as expected to generate a password with a special character with # or *

Reply:
‘*’ causes a problem in generating password. I changed it to '@' and tested it.